fast_response/MonitoringAndMocks/Data_Display.py

```diff
@@ -8,756 +9,392 @@
 matplotlib.rcParams['ytick.labelsize'] = 16
 import matplotlib.pyplot as plt
 import numpy as np
-import glob, os, datetime, pwd, pickle
-from datetime import date
+import pickle
+import glob
+import os
+import datetime
+from datetime import date, timezone
+from dateutil.relativedelta import *
 from statistics import median
-#from statistics import mode
 from matplotlib.dates import DateFormatter
 from astropy.time import Time
 import pandas as pd
+# from IPython.display import HTML
+# from IPython.display import display
 import subprocess
-import warnings
-warnings.filterwarnings('ignore', module='astropy._erfa')
 
 def dial_up(who="jessie"):
```

Comment:
Can you switch to this path? had to move the file
```cell_tower = "/home/jthwaites/private/"```
and if you can remove the commented out call file on the next line, that would be good too